sqlx-core/Cargo.toml

```diff
@@ -46,8 +47,8 @@ runtime-tokio-rustls = [ "sqlx-rt/runtime-tokio-rustls", "_tls-rustls", "_rt-tok
 _rt-actix = []
 _rt-async-std = []
 _rt-tokio = []
-_tls-native-tls = []
-_tls-rustls = [ "rustls", "webpki", "webpki-roots" ]
+_tls-native-tls = [ "rusoto_core/native-tls", "rusoto_rds_data/native-tls" ]
+_tls-rustls = [ "rustls", "webpki", "webpki-roots", "rusoto_core/rustls", "rusoto_rds_data/rustls" ]
```

Comment:
It seems like this causes the deps to be included, even if "aurora" isn't. It seems there is a tracking issue for weak dependencies that could solve this. Is there another workaround so we don't have 2 feature gates for aurora... "aurora-native-tls" / "aurora-rustls"?

Reply:
Cross-reference: https://github.com/rust-lang/cargo/pull/8818